doc/HttpLuaModule.wiki

```diff
@@ -565,24 +565,23 @@ will not work as expected.
 
 == Cosockets Not Available Everywhere ==
 
-Due the internal limitations in the nginx core, the cosocket API are disabled in the following contexts: [[#set_by_lua|set_by_lua*]], [[#log_by_lua|log_by_lua*]], [[#header_filter_by_lua|header_filter_by_lua*]], and [[#body_filter_by_lua|body_filter_by_lua]].
+Due the internal limitations in the nginx core, the cosocket API is disabled in the following contexts: [[#set_by_lua|set_by_lua*]], [[#log_by_lua|log_by_lua*]], [[#header_filter_by_lua|header_filter_by_lua*]], and [[#body_filter_by_lua|body_filter_by_lua]].
```

Comment:
typo: "due **to** internal limitations in **the** nginx core"

Additionally, most occurrences of "Nginx" in this document are capitalizes (`Nginx` vs `nginx`), except in a few places, like this one. Should this be capitalized?